unattended-upgrade

```diff
@@ -721,10 +745,16 @@ def conffile_prompt(destFile, prefix=""):
     # get conffile value from pkg, its ok if the new version
     # does not have conffiles anymore
     pkg_conffiles = ""
-    deb = apt_inst.DebFile(destFile)
     try:
+        deb = apt_inst.DebFile(destFile)
         pkg_conffiles = deb.control.extractdata("conffiles").strip().decode(
             "utf-8")
+    except SystemError as e:
+        print(_("Apt returned an error, exiting"))
+        print(_("error message: '%s'") % e)
```

Comment:
Maybe remove this print and just `raise` again at the `sys.exit(1)` point?

Reply:
For the users raising the issue would be as informative as printing it, but I'd like to avoid reporting known and handled errors to errors.ubuntu.com.